### PR TITLE
General Refactor

### PR DESCRIPTION
**Pull request by SeoFernando25**
- Reorganized [import order](https://www.python.org/dev/peps/pep-0008/#:~:text=Imports%20should%20be%20grouped%20in%20the%20following%20order)
- Refactor on functions and variable names

[Note on autoescape](https://github.com/SeoFernando25/krait/blob/bc7a689158f44fd3d43f774c9bea3a3912f35b71/src/krait/utils/templates.py#L5): Something worth noting Jinja2 environment creation is that `autoescape` is set to `False` by default, would it do any harm in long term to turn it on to something like `autoescape=select_autoescape(['html', 'xml'])`?

### CODE REVIEW

File: src/krait/utils/update.py [hunk 1]
```diff
@@ -69,14 +70,12 @@ def should_check_update(ctx: click.Context) -> bool:
...
+        return subprocess.check_output([
             sys.executable,
             *cmd.split()
         ], stderr=subprocess.DEVNULL).decode('utf-8')
-    except Exception:
-        o = ''
-
-    return o
+    except subprocess.CalledProcessError:  # Non-zero exit code
+        return ''
```
Comment (taliamax) [inline src/krait/utils/update.py:78]: The more general exception was purposeful here; want to ensure that any other issue that occurs gets caught (like decoding issues etc) and don't get relayed back to user

File: src/krait/utils/update.py [hunk 2]
```diff
@@ -116,12 +115,6 @@ def run_update():
...
-            sys.executable,
-            *'-m pip install --user --upgrade krait'.split()
-        ])
+        run_pip('install --user --upgrade krait')
     else:
-        subprocess.call([
-            sys.executable,
-            *'-m pip install --upgrade krait'.split()
-        ])
+        run_pip('install --upgrade krait')
```
Comment (taliamax) [inline src/krait/utils/update.py:120]: `run_pip` uses subprocess.check_output, subprocess.call was chosen here to just act as a pass-through while still throwing output out to stdout

File: .gitignore [hunk 3]
```diff
@@ -135,3 +135,6 @@ dmypy.json
 
 # Cython debug symbols
 cython_debug/
+
+# VSCode user settings
+.vscode/
```
Comment (taliamax) [inline .gitignore:140]: Don't mind this being added here, but it should go in your global ignore: http://egorsmirnov.me/2015/05/04/global-gitignore-file.html

File: src/krait/cli.py [hunk 4]
```diff
@@ -54,8 +54,8 @@
 
 help_links = {}
 
-for obj in help_link_objects.values():
-    help_links.update(obj.links)
+for value in help_link_objects.values():
+    help_links.update(value.links)
```
Comment (taliamax) [inline src/krait/cli.py:58]: don't mind the name change but I'm not sure 'value' is appropriate. I'll give this some more thought and address this in a later change

File: src/krait/utils/templates.py [hunk 5]
```diff
@@ -2,4 +2,4 @@
 import jinja2
 
 
-env = jinja2.Environment(loader=jinja2.PackageLoader('krait', 'templates'))
+env = jinja2.Environment(loader=jinja2.PackageLoader('krait', 'templates'), autoescape=False)
```
Comment (taliamax) [inline src/krait/utils/templates.py:5]: unsure about the implications of not auto escaping, though that's more of a note for myself later on to check

**Review by taliamax — CHANGES_REQUESTED** [review]
Looks really good! Just a couple of minor changes on the update utilities. Feel free to up the micro version of the release!

File: src/krait/cli.py (hunk 4 above)
Comment (SeoFernando25) [reply src/krait/cli.py:58]: Oops, this was just some repeated variable warning that showed up on my linter due to the variable `obj` already being used in a function declaration.

https://github.com/taliamax/krait/blob/71e6d21719d50dcffd0e764506d0df519c75b70c/src/krait/cli.py#L458

I guess it's just my linter being weird... my bad

**Pull request merged**